public/index.html

```diff
@@ -16,8 +16,8 @@
     <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="theme-color" content="#ffffff">
-
     <title>Mesh</title>
+    <title><%= htmlWebpackPlugin.options.title %></title>
```

Comment:
I'd love to understand more about how these title tags are working! I thought the `<%= htmlWebpackPlugin.options.title %>` line that was there before was dynamically adding the title somehow? But maybe we need both?

Reply:
I've never really looked this up, but it seems that there are a number of ways to do it:  https://stackoverflow.com/questions/62023604/where-to-find-or-how-to-set-htmlwebpackplugin-options-title-in-project-created-w